openlibrary/templates/type/author/view.html

```diff
@@ -68,15 +68,19 @@
 
     $:macros.databarView(page)
 
-    <h1>$title</h1>
-    <h2 class="author collapse">
-    $if page.birth_date or page.death_date:
-        $page.birth_date - $page.death_date
-    $else:
-        $if page.date:
-            $page.date
-    </h2>
+    <div itemscope itemtype="https://schema.org/Person">
+        <h1 itemprop="author">$title</h1>
```

Comment:
Author isn't expected on type `Person`:

```suggestion
        <h1 itemprop="name">$title</h1>
```

Reply:
I was confused about this, thanks for clearing it out. will make the required changes.